<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-HMIG7V7" failures="1" tests="4" name="PetTest" time="12.465" errors="0" timestamp="2025-03-19T17:58:32 IST">
  <testcase classname="api.testcase.UserTests2" name="testPostUser" time="5.529"/>
  <testcase classname="api.testcase.UserTests2" name="testGetUserByName" time="1.453"/>
  <testcase classname="api.testcase.UserTests2" name="testUpdateUserByName" time="2.874">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.testcase.UserTests2.testUpdateUserByName(UserTests2.java:76)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- testUpdateUserByName -->
  <testcase classname="api.testcase.UserTests2" name="testDeleteUserByName" time="1.392"/>
</testsuite> <!-- PetTest -->
